src/notifier/discord_notifier.py

```diff
@@ -4,16 +4,19 @@
 import urllib.parse
 from typing import List
 
+# lib
+from confuse import ConfigView
+
 # project
 from . import Notifier, Event
 
 
 class DiscordNotifier(Notifier):
-    def __init__(self, title_prefix: str, config: dict):
+    def __init__(self, title_prefix: str, config: ConfigView):
         logging.info("Initializing Discord notifier.")
         super().__init__(title_prefix, config)
         try:
-            credentials = config["credentials"]
+            credentials = config["credentials"].get(dict)
```

Comment:
Unfortunately, I don’t have a setup to run more comprehensive tests on the notifiers. Were you able to successfully run one of them which included this `get(dict)` parsing to make sure credentials are generally getting parsed in backward compatible way with the new library?